src/pages/_app.tsx

```diff
@@ -4,6 +4,7 @@ import NextApp, { AppContext } from 'next/app';
 import React from 'react';
 import { Provider } from 'react-redux';
 import makeStore from '../store';
+import '../styles/index.css';
```

Comment:
Nothing to change here for now just a note that this should be added to the aliases in the future.

Reply:
@DeanMW we can potentially do that, however, this file should only ever be imported in the `_app.tsx` file, so we'll never actually import the global styles anywhere else